mirascope/core/base/from_call_args.py

```diff
@@ -0,0 +1,36 @@
+from pydantic.fields import FieldInfo
+
+
+class FromCallArgs:
+    """Use this metadata to pass function arguments through to the response model.
+
+    Example:
+
+    ```python
+    from typing import Annotated
+    from pydantic import BaseModel, Field
+    from mirascope.core import openai, prompt_template
+    from mirascope.core.base import FromCallArgs
+
+    class BookRecommendation(BaseModel):
+        genre: Annotated[str, FromCallArgs()]
```

Comment:
It works as a function, but may not be the optimal use case.

Reply:
what do you mean?

Reply:
There may be better examples, but we used genre to match other examples.